docs/dev/core/internal/implementations.md

```diff
@@ -0,0 +1,17 @@
+# Implementations
+
+| Project Name      | Language Bindings   | Operating System  | App Lifecycle Type  | Environment Data source |
+| ----------------- | ------------------- | ----------------  | ------------------- | ----------------------- |
+| glean-core        | Rust                | all               | all                 | none                    |
+| glean-ffi         | C                   | all               | all                 | none                    |
```

Comment:
Are we ok with calling out the FFI binding here?
As I said in the proposal doc we don't advertise this as supported, I still think listing it in internal docs is ok.

Reply:
I think it's fine,it's internal docs anyway.